LeMinhQuoc/blog-frontend
# Feature/list blog

- Solve : display listBlog
- How  dose it PR solve it
      + Get data from api/blogs 
      + Render value to Home/index.twig 
      + Loop listBlog and return in font-end
- How can reviewer test the changes
     + Pull this branch on your local.
     + Install [API source](https://github.com/LeMinhQuoc/blog-api)
     + Run composer install.
     + Run symfony server:start
     + Flow https://localhost/ to check
